### ISSUE TITLE
Missing fields

### ISSUE THREAD

**Issue by m-vo** [derhaeuptling/contao-immobilienscout24-import-bundle#4]
`RealEstateEntity`

```
apartmentType
floor
assistedLiving
industryType
hallHeight
flooringType
ramp
lift
autoLift
goodsLift
goodsLiftLoad
craneRunway
craneRunwayLoad
floorLoad
connectedLoad
investmentType
pricePerSQM::value
pricePerSQM::currency
pricePerSQM::marketingType
pricePerSQM::priceIntervalType
priceMultiplier
rentalIncomeActual::value
rentalIncomeActual::currency
rentalIncomeActual::marketingType
rentalIncomeActual::priceIntervalType
otherCosts::value
otherCosts::currency
otherCosts::marketingType
otherCosts::priceIntervalType
officeType
hasCanteen
lanCables
highVoltage
kitchenComplete
airConditioning
officeRentDurations
baseRent
totalRent
serviceCharge
deposit
heatingCosts
heatingCostsInServiceCharge
petsAllowed
useAsFlatshareRoom
price::value
price::currency
certificateOfEligibilityNeeded
garageType
lengthGarage
widthGarage
heightGarage
commercializationType
totalFloorSpace
netFloorSpace
minDivisible
distanceToMRS
distanceToFM
distanceToPT
distanceToAirport
freeFrom
recommendedUseTypes
tenancy
shortTermConstructible
buildingPermission
demolition
siteDevelopmentType
siteConstructibleType
grz
gfz
leaseInterval
basement
storeType
locationClassificationType
shopWindowLength>6.151</shopWindowLength>
supplyType>FORENOON</supplyType>
additionalCosts::value
additionalCosts::currency
additionalCosts::marketingType
additionalCosts::priceIntervalType
utilizationTradeSite
energyCertificate::electricityConsumption
energyCertificate::heatingConsumption
energyCertificate::energyConsumptionElectricity
energyCertificate::energyConsumptionHeating
energyCertificate::thermalCharacteristicElectricity
energyCertificate::thermalCharacteristicHeating
energySourcesEnev2014
```

**Comment by m-vo** [comment]
@Metis77 These would be **87** additional fields (to the already 59 in `RealEstateEntity`). Could you please define a small criticial subset of those fields that need to be implemented now?

**Comment by Metis77** [comment]
Problem is, that I do not know for sure.
But I will give a guess on what I think now.



**apartmentType**
floor
assistedLiving
**industryType**
hallHeight
flooringType
ramp
**lift**
autoLift
goodsLift
goodsLiftLoad
craneRunway
craneRunwayLoad
floorLoad
connectedLoad
**investmentType**
pricePerSQM::value
pricePerSQM::currency
pricePerSQM::marketingType
pricePerSQM::priceIntervalType
priceMultiplier
rentalIncomeActual::value
rentalIncomeActual::currency
rentalIncomeActual::marketingType
rentalIncomeActual::priceIntervalType
otherCosts::value
otherCosts::currency
otherCosts::marketingType
otherCosts::priceIntervalType
**officeType**
hasCanteen
lanCables
highVoltage
kitchenComplete
airConditioning
**officeRentDurations**
**baseRent**
**totalRent**
serviceCharge
deposit
**heatingCosts**
**heatingCostsInServiceCharge**
**petsAllowed**
useAsFlatshareRoom
**price::value
price::currency**
certificateOfEligibilityNeeded
**garageType**
lengthGarage
widthGarage
heightGarage
**commercializationType**
totalFloorSpace
netFloorSpace
minDivisible
distanceToMRS
distanceToFM
distanceToPT
distanceToAirport
**freeFrom**
**recommendedUseTypes**
tenancy
shortTermConstructible
**buildingPermission**
demolition
siteDevelopmentType
siteConstructibleType
grz
gfz
leaseInterval
**basement**
**storeType**
locationClassificationType
shopWindowLength>6.151</shopWindowLength>
supplyType>FORENOON</supplyType>
additionalCosts::value
additionalCosts::currency
additionalCosts::marketingType
additionalCosts::priceIntervalType
**utilizationTradeSite**
energyCertificate::electricityConsumption
energyCertificate::heatingConsumption
energyCertificate::energyConsumptionElectricity
energyCertificate::energyConsumptionHeating
energyCertificate::thermalCharacteristicElectricity
energyCertificate::thermalCharacteristicHeating


Update: I removed some more fields.

**Comment by m-vo** [comment]
implemented in d6554170263a3a0dae019a25160cbb01740e933b
